src/controllers/OtherReasonController.ts

```diff
@@ -32,7 +32,7 @@ export class OtherReasonController extends SafeNavigationBaseController<OtherRea
     }
 
     protected prepareViewModelFromAppeal(appeal: Appeal): Record<string, any> & OtherReason {
```

Comment:
Can update the return types?

Reply:
It is returning OtherReason it should not be changed to something else